src/test/java/io/pravega/connectors/flink/FlinkPravegaTableITCase.java

```diff
@@ -312,8 +332,13 @@ private void testTableSourceBatchDescriptor(Stream stream, PravegaConfig pravega
         final Map<String, String> propertiesMap = desc.toProperties();
         final TableSource<?> source = TableFactoryService.find(BatchTableSourceFactory.class, propertiesMap)
                 .createBatchTableSource(propertiesMap);
+        ConnectorCatalogTable<?, ?> connectorCatalogSourceTable = ConnectorCatalogTable.source(source, false);
```

Comment:
The old API `registerTableSource` API can still work here, it is correct. We can revert the change here, and found the best way for Blink planner to create the table after fixing #356  . We can revert the document change for this, too